src/views/Hello.vue

```diff
@@ -1,26 +1,36 @@
 <template>
-    <h1>
-        {{ message }}
-    </h1>
+    <v-app>
+        <v-app-bar app>
+            <v-toolbar-title>
+                {{ title }}
+            </v-toolbar-title>
+        </v-app-bar>
+        <v-content>
+            <v-container>
+                <v-btn
+                    fixed
+                    fab
+                    bottom
+                    right
+                    color="primary"
+                >
+                    <v-icon>
+                        mdi-plus
+                    </v-icon>
+                </v-btn>
+            </v-container>
+        </v-content>
+    </v-app>
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
-import { Component } from 'vue-property-decorator';
```

Comment:
Does vue-decorator not work with vuetify ?

Reply:
Yes it does. I shouldn't have remove it.